packages/augur-ui/src/modules/notifications/selectors/notification-state.ts

```diff
@@ -94,17 +94,22 @@ export const selectMarketsInDispute = createSelector(
   selectAccountPositionsState,
   selectLoginAccountAddress,
   (markets, positions, address) => {
+    const state = store.getState();
+    const disputedMarkets = state
+      .loginAccount.reporting.disputing.contracts.map(obj => obj.marketId);
```

Comment:
either one of these can be null. 
state.loginAccount.reporting.reporting
state.loginAccount.reporting.disputing

also, I don't see markets where user has created.